<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogementsWindow</class>
 <widget class="QMainWindow" name="LogementsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1209</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton
{
 background :  ; 
 border-radius : 60px ; 
}
QLabel
{
 color : #48494B  ; 
}
QLineEdit
{
 background:transparent ; 
border : none ;
color:#48494B ;
font-size : 20px ; 
border-bottom:0.5px solid #717072 ; 
}
QTabWidget::pane {
  top:-40px; 
} 
QTabBar::tab {
background-color: rgb(217, 221, 220);
border-radius : 15px;
border: 2px solid lightgray; 
border-color: rgb(255, 255, 255);
padding: 3px;
} 

QTabBar::tab:selected { 
background-color: rgb(217, 221, 220);
border: 3px solid lightgray; 
border-color: rgb(255, 255, 255);
border-bottom-color: rgb(255, 140, 0);
border-right-color: rgb(255, 140, 0);
margin-bottom: 0px; 

}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1121</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 11pt &quot;Montserrat SemiBold&quot;;
background-color: rgb(217, 221, 220);
border-radius : 35px ; </string>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab_Create_Housing">
     <attribute name="title">
      <string>Create Housing</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>40</y>
        <width>221</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 36pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Housing</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>200</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>270</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Housing Condition :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>200</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Icons/outil.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>550</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>200</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>id</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>280</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>etat</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_logo">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>160</y>
        <width>261</width>
        <height>251</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../Icons/ApplicationHomeShare.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>270</y>
        <width>261</width>
        <height>251</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Indigents/Icons/ApplicationHomeShare.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_HousingManagment">
     <attribute name="title">
      <string>Housing Managment</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>50</y>
        <width>361</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 36pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>List Housing</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_somme">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>200</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}
</string>
      </property>
      <property name="text">
       <string>somme</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Search">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>220</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_update">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>480</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}
</string>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QLabel" name="Icon_Search_label">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>220</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Fournisseurs/Icons/search.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_pdf">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>200</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}
</string>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_Delete">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>480</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}
</string>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_list">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>280</y>
        <width>541</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>320</y>
        <width>221</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(243, 251, 255)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stat">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>200</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}
</string>
      </property>
      <property name="text">
       <string>statistics</string>
      </property>
     </widget>
     <widget class="QPushButton" name="historique">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>200</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}
</string>
      </property>
      <property name="text">
       <string>historique</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Statistics">
     <attribute name="title">
      <string>somme</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit0">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>230</y>
        <width>531</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>250</y>
        <width>261</width>
        <height>251</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Indigents/Icons/ApplicationHomeShare.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>921</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 36pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>somme des logements selon l'etat:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>update</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>70</y>
        <width>221</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 36pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Housing</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>200</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Icons/outil.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_adresse_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>200</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_2">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>200</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>id</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>300</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 14pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Housing Condition :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_condition_2">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>300</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>etat</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>550</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background : #FF8C00 ; 
 border-radius : 25px ; 
font-size : 20px ;
color : #48494B ;  
}
QPushButton:hover
{
 background : #48494B; 
 border-radius : 25px ; 
color : white ;  
}</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>261</width>
        <height>251</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Indigents/Icons/ApplicationHomeShare.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>stat</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>230</y>
        <width>761</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>60</y>
        <width>621</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 36pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>statistics of housing</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>260</y>
        <width>261</width>
        <height>251</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Indigents/Icons/ApplicationHomeShare.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>historique</string>
     </attribute>
     <widget class="QListView" name="listView_historique">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>240</y>
        <width>951</width>
        <height>381</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>10</y>
        <width>201</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Indigents/Icons/ApplicationHomeShare.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>60</y>
        <width>361</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 63 36pt &quot;Montserrat SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>historique</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1209</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
